### PR TITLE
add the default `cube` scene to alleviate issues with the JPEG integration tests

### PR DESCRIPTION
**Pull request by shadeofblue**
(the scene's simpler lighting and less abrupt transitions should make the compression artifacts less prominent and the failures thus rarer)

### CODE REVIEW

**Review by maaktweluit — APPROVED** [review]
LGTM!

Maybe we can replace `cubes` with `cube` as default if it gives more consistent results

**Review by jiivan — APPROVED** [review]

File: scripts/node_integration_tests/tests/test_golem.py [hunk 1]
```diff
@@ -56,7 +56,10 @@ def test_jpeg(self):
         verifies if Golem - when supplied with `JPEG` as the format - will
         render the output as JPEG with the proper extension.
         """
-        self._run_test('golem.jpeg')
+        self._run_test(
+            'golem.regular_run_stop_on_reject',
```
Comment (etam) [inline scripts/node_integration_tests/tests/test_golem.py]: you changed the test here, so you're no longer testing jpeg

**Comment by shadeofblue** [comment]
> LGTM!
> 
> Maybe we can replace `cubes` with `cube` as default if it gives more consistent results

I wouldn't do that - especially that other formats don't have this problem... 

(we don't want to test everything on a default scene)

File: scripts/node_integration_tests/tests/test_golem.py (hunk 1 above)
Comment (shadeofblue) [reply scripts/node_integration_tests/tests/test_golem.py]: [ facepalm ]

File: scripts/node_integration_tests/tests/test_golem.py [hunk 2]
```diff
@@ -56,7 +56,10 @@ def test_jpeg(self):
         verifies if Golem - when supplied with `JPEG` as the format - will
         render the output as JPEG with the proper extension.
         """
-        self._run_test('golem.jpeg')
+        self._run_test(
+            'golem.jpg',
```
Comment (etam) [inline scripts/node_integration_tests/tests/test_golem.py]: No, it's `jpeg`. The `e` in the middle makes a difference.

Comment (etam) [reply scripts/node_integration_tests/tests/test_golem.py]: `jpg` is a different test

**Review by etam — APPROVED** [review]
ok

File: scripts/node_integration_tests/tests/test_golem.py (hunk 2 above)
Comment (maaktweluit) [reply scripts/node_integration_tests/tests/test_golem.py]: `jpeg` is the correct output type, rendering into `.jpg` extension
https://github.com/golemfactory/golem/blob/develop/apps/blender/resources/images/entrypoints/scripts/verifier_tools/file_extension/types.py#L53

while `jpg` is an unsupported output type thus defaulting to PNG
https://github.com/golemfactory/golem/blob/develop/apps/blender/task/blenderrendertask.py#L247

maybe `jpg` should be changed to `abc` to be more clear.

**Pull request merged**